lib/macho/macho_file.rb

```diff
@@ -424,6 +463,16 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
+      fmt = Utils.specialize_format("L=", endianness)
+      new_ncmds = [ncmds].pack(fmt)
```

Comment:
`ncmds_bytes`, `ncmds_raw`, or something similar would be a better name. The `ncmds` variable already contains the new value, so that's somewhat confusing currently.